Harvest.Core/Services/SlothService.cs

```diff
@@ -94,6 +94,14 @@ public async Task<Result<SlothResponseModel>> MoveMoney(int invoiceId)
                 invoice.Total = grandTotal;
             }
 
+            if (invoice.Expenses.Any(a => a.Total < 0))
+            {
+                if (!(ProcessRefunds(model, grandTotal, invoice)).Value)
+                {
+                    return Result.Error("ProcessRefunds Failed");
+                }
```

Comment:
I think this will swallow the error -- perhaps it's better to get back the Result<bool> and if it's .Value is false instead of creating a new error just return the error that you get back?

Reply:
ah so result.Error as well as being a return value also immediately logs?  interesting didn't know that.